modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -48,23 +47,25 @@ export interface ICreateProposalParams {
   endDate?: Date;
   executeOnPass?: boolean;
   creatorVote?: VoteValues;
+  voteReplacement?: boolean;
+  earlyExecution?: boolean;
 }
 
 export interface IVoteProposalParams {
   pluginAddress: string;
   vote: VoteValues;
-  proposalId: string;
+  proposalId: bigint;
 }
 
 export interface IExecuteProposalParams {
   pluginAddress: string;
-  proposalId: string;
+  proposalId: bigint;
 }
 
 export interface ICanVoteParams {
   pluginAddress: string;
-  proposalId: string;
-  address: string;
+  proposalId: bigint;
+  addressOrEns: string;
```

Comment:
As discussed before, this should be a single hex string value, everywhere

Reply:
Yes, the problem is that if we specify the new format `0x...{64}` we won’t be passing tests because neither the contracts nor the subgraph is updated